runtime.md

```diff
@@ -9,8 +9,11 @@ $ funC [global-options] <COMMAND> [command-specific-options] <command-specific-a
 
 ## Global options
 
-None are required, but the runtime may support options that start with at least one hyphen.
-Global options may take positional arguments (e.g. `--log-level debug`), but the option parsing must be such that `funC <COMMAND>` is unambiguously an invocation of `<COMMAND>` for any `<COMMAND>` that does not start with a hyphen (including commands not specified in this document).
+None are required, but the runtime MAY support options that start with at least one hyphen.
+Global options MAY take positional arguments (e.g. `--log-level debug`).
+Command names MUST not start with hyphens.
```

Comment:
On Mon, Dec 07, 2015 at 03:54:55PM -0800, Mike Brown wrote:
> > +Command names MUST not start with hyphens.
> 
> You're saying -START is not the same as START wrt compliance.

No, I'm saying:

  The only commands we define now are ‘version’, ‘start’, ‘exec’,
  ‘pause’, ‘resume’, and ‘signal’.  We may release a future version of
  this spec with a command like ‘checkpoint’, but will not release any
  future versions of this spec with a command like ‘-checkpoint’.

Without a pattern like “commands will not start with a hyphen” it
would be harder to distinguish unrecognized commands from unrecognized
options, and we're requiring runtimes to fail-fast for unrecognized
commands.

I agree that ‘-start’ and ‘start’ are not the same, but I think that's
clear without this sentence.